pkg/minikube/node/cache.go

```diff
@@ -99,14 +99,20 @@ func doCacheBinaries(k8sVersion string) error {
 }
 
 // BeginDownloadKicArtifacts downloads the kic image + preload tarball, returns true if preload is available
-func beginDownloadKicArtifacts(g *errgroup.Group) {
+func beginDownloadKicArtifacts(g *errgroup.Group, driver string, cRuntime string) {
 	glog.Info("Beginning downloading kic artifacts")
-	if !image.ExistsImageInDaemon(kic.BaseImage) {
-		out.T(out.Pulling, "Pulling base image ...")
-		g.Go(func() error {
-			glog.Infof("Downloading %s to local daemon", kic.BaseImage)
-			return image.WriteImageToDaemon(kic.BaseImage)
-		})
+	glog.Infof("Beginning downloading kic artifacts for %s with %s", driver, cRuntime)
+	if driver == "docker" {
+		if !image.ExistsImageInDaemon(kic.BaseImage) {
+			out.T(out.Pulling, "Pulling base image ...")
+			g.Go(func() error {
+				glog.Infof("Downloading %s to local daemon", kic.BaseImage)
+				return image.WriteImageToDaemon(kic.BaseImage)
+			})
+		}
+	} else {
+		// TODO: driver == "podman"
+		glog.Info("Container runtime isn't docker, skipping download")
```

Comment:
was this supposed to say driver or container run time ?
because the if checks for driver, but else says runtime.

Reply:
We can change this, but it is only for the internal logging anyway.

Much better would be to fix it, by dropping go-containerregistry ?